README.md

```diff
@@ -28,17 +28,15 @@ Gupax may not run on machines with:
 * [License](#license) 
 
 ## What is Gupaxx/XMRvsBeast?
-[**`Gupaxx`**](https://getmonero.org) is a fork of [*Gupax*](https://github.com/hinto-janai/gupax) that integrate the [XMRvsBeast raffle](https://xmrvsbeast.com).
+[**`Gupaxx`**](https://getmonero.org) is a fork of [*Gupax*](https://github.com/hinto-janai/gupax) that integrates the [XMRvsBeast raffle](https://xmrvsbeast.com).
 
-This fork adds an easy way to distribute your hashrate between P2pool and XMRvsBeast to participate in the raffle to win more xmr while supporting the p2pool network.
+With this fork, you can easily split your hashrate between P2Pool and XMRvsBeast, increasing your chances of winning in the raffle while also supporting the P2Pool ecosystem.
```

Comment:
P2Pool ecosystem seems weird for me. But supporting P2Pool network is not really the goal, so we can rephrase it with "supporting the Monero network by decentralizing the mining by using p2pool", or a better sentence with the same sense.

Reply:
changed:
while also supporting the P2Pool ecosystem
to:
while also supporting the Monero network via decentralizing the mining using using p2poo